src/chttpd/src/chttpd_view.erl

```diff
@@ -91,19 +91,19 @@ handle_view_req(#httpd{method='POST',
     Keys = couch_mrview_util:get_view_keys(Props),
     Queries = couch_mrview_util:get_view_queries(Props),
     case {Queries, Keys} of
```

Comment:
rather than keep the 2-tuple and have it always been an error to specify the first one, I think it would be cleaner to have if clauses for the two errors (Queries not being undefined, Keys not being a list) and then the two lines of actual implementation?